keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -301,7 +316,7 @@ kind: BucketClass
 metadata:
   name: 
 provisioner: [1]
-supportedProtocols: [2]
+protocol: [2]
```

Comment:
Do also want the `BucketContentIdentity` in the `BucketClass`? Or will fields like _accountName_ be defined in _parameters_?

Reply:
accountName i would think are returned by the driver.  we can't determine if a driver would provisioner new accounts per BucketContent, or a new key in a meta-account.